crates/uv-install-wheel/src/wheel.rs

```diff
@@ -186,6 +187,25 @@ pub(crate) fn write_script_entrypoints(
     is_gui: bool,
 ) -> Result<(), Error> {
     for entrypoint in entrypoints {
+        let warn_names = ["activate", "activate_this.py"];
+        if warn_names.contains(&entrypoint.name.as_str())
+            || entrypoint.name.starts_with("activate.")
+        {
+            warn_user_once!(
+                "The script name `{}` is reserved for virtual environment activation scripts.",
+                entrypoint.name
+            );
```

Comment:
Why a warning here rather than an error? (Just wondering.)

Reply:
We could also error here, but they didn't feel critical enough